projects/imminence/Makefile

```diff
@@ -1,3 +1,3 @@
-imminence: bundle-imminence mapit
+imminence: bundle-imminence locations-api
```

Comment:
Ah, does Imminence have a dependency on Locations API? I thought you'd discovered it should only have a dependency on Local Links Manager instead?

Reply:
Updated.